GCDWebServer.xcodeproj/project.pbxproj

```diff
@@ -861,13 +861,14 @@
 					};
 					CEE28CEE1AE0051F00F4023C = {
 						CreatedOnToolsVersion = 6.3;
-						DevelopmentTeam = 88W3E55T4B;
+						ProvisioningStyle = Manual;
```

Comment:
for dynamic frameworks in iOS, signing should be disabled. For static frameworks in project, I'm not sure